jplag/src/main/java/de/jplag/clustering/ClusteringResult.java

```diff
@@ -48,7 +48,7 @@ public float getCommunityStrength() {
 
     /**
      * How much this clustering result is worth during optimization.
-     * @param similarity
+     * @param similarity TODO JAVADOC
```

Comment:
We use name abbreviations with TODO comments. For example my comments look like this: `//TODO TS: Do something`

Reply:
or is it more about who defined the TODO comment :)

Reply:
The latter, whoever points out these issues. Fixing them is another thing...

Reply:
```suggestion
     * @param similarity TODO DF: JAVADOC
```